README.md

```diff
@@ -23,23 +23,28 @@ pip install personal-graph
 ### Building a Working Memory for an AI
 
 ```python
-from personal_graph import Graph
+from personal_graph.graph import Graph
```

Comment:
`from personal_graph import Graph` this should work fine

Reply:
DONE